packages/mobile/src/screens/profile-screen/ProfileHeader/SocialLink.tsx

```diff
@@ -175,3 +180,29 @@ export const TikTokSocialLink = (props: TikTokSocialLinkProps) => {
     />
   )
 }
+
+export const WebsiteSocialLink = (props: Partial<SocialLinkProps>) => {
+  const { website } = useSelectProfile(['website'])
```

Comment:
nit: Feels kinda unnecessary to select here when we could just pass the value in from above? Select stuff like this feels like it moves _away_ from tan-querying most of our data.

Reply:
yeah i hear that, just following the pattern with the other ones here, where they are used in two places (collapsed and expanded section)